styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -76,6 +76,18 @@ img { max-width: 100%; }
   margin: 0 auto;
 }
 
+// Horizontal rules
+hr {
```

Comment:
@WidgetsBurritos The hr I'm adding has a class. The only edit there was to the existing hr was to reduce the margin around it, which we decided needed to happen to all the hrs. Since the margin is straight crazy. Would you suggest instead of adding an hr class to add like div class="hr-red" or something? I'm a little wary to start calling things a div with a class of an hr. Since it gets a little complex when we're trying to actually put them on the page.